data/templates/ssh/sshd_config

```diff
@@ -94,3 +87,11 @@ Match User sftpusers
         AllowTcpForwarding no
         GatewayPorts no
         X11Forwarding no
+
+# root login is allowed on local networks
+# It's meant to be a backup solution in case LDAP is down and
+# user admin can't be used...
+# If the server is a VPS, it's expected that the owner of the
+# server has access to a web console through which to log in.
+Match Address 192.168.0.0/16,10.0.0.0/16
```

Comment:
Is it not 10.0.0.0/8 and are there not also 172.16.0.0/12
And for IPv6 ? Didn't test but maybe fe80::/16 could be also useful :smile:

Reply:
done, i added link local too